tests/FileLockTest.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+$fname = 'tests/lock-test.txt';
+
+//create file
+$handle = fopen($fname, 'w') or die('ERROR[0]: Unable to open/create file "'.$fname.'"!');
+$txt = date('d.m.Y H:i:s');
+fwrite($handle, $txt);
+fclose($handle); 
+echo PHP_EOL .'Test file "'.$fname.'" created ['.date('d.m.Y H:i:s').']. FileSize: '.filesize($fname).' bytes.'. PHP_EOL . PHP_EOL; 
+
+
+class FileLockTest extends TestCase {
+
+	public function testCanBeLocked() {
+
+		global $fname;
+		//create lock object
+		$lock = new \Dorantor\FileLock($fname);
+		
+		//try to lock file
+        $this->assertEquals(true, $lock->acquire(),'ERROR[1]: Result of function acquire() not correct!'. PHP_EOL);
+/*		
```

Comment:
Why did you commit code that is commented out?

Reply:
Actually Git do it in automatic way after i add Travis CI to my repository. And i sow this so late after i do a lot of commits.